CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
+    date version of the plugin it has been removed from Packer. To install the
+    latest version of the plugin please run `packer plugins install
+    github.com/outscale/outscale`.
+    [GH-11912](https://github.com/hashicorp/packer/pull/11912)
+* packer-plugin-scaleway: The Scaleway plugin has been maintained by the
+    Scaleway team for some time now. In order to ensure users the most up to
+    date version of the plugin it has been removed from Packer. To install the
+    latest version of the plugin please run `packer plugins install
+    github.com/scaleway/scaleway`.
+    [GH-11912](https://github.com/hashicorp/packer/pull/11912)
+
+### FEATURES:
+* Future Scaffolding: This release contains additional changes that allow
+    Packer core to validate that a newly built image is a direct child of a HCP
+    Packer registry source image. This feature is only available for HCP Packer
+    enabled builds using the `hcp_packer_image` and `hcp_packer_iteration` data
+    source for setting a builder's source image.
+    [GH-11861](https://github.com/hashicorp/packer/pull/11861)
+
+### PLUGINS:
+
+The following external plugins have been updated and pinned to address open
+    issues. Please see their respective changelogs for details on plugin
+    specific bug fixes and improvements.
+
+* amazon@v1.1.2 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-amazon/releases/tag/v1.1.2)
+* ansible@v1.0.3 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-ansible/releases/tag/v1.0.3)
+* azure@v1.3.0 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-azure/releases/tag/v1.3.0)
+* docker@v1.0.7 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-docker/releases/tag/v1.0.7)
+* googlecompute@v1.0.14 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-googlecompute/releases/tag/v1.0.14)
+* lxc@v1.0.2 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-lxc/releases/tag/v1.0.2)
+* triton@v1.0.2 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-triton/releases/tag/v1.0.2)
+* vsphere@v1.0.7 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-vsphere/releases/tag/v1.0.7)
+* yandex@v1.1.2 - [CHANGELOG](https://github.com/hashicorp/packer-plugin-yandex/releases/tag/v1.1.2)
+
+
+### IMPROVEMENTS:
+* Bump packer-plugin-sdk to v0.3.1 to fix inconsistencies between `ssh_timeout`
+    and `ssh_handshake_attempts` configuration arguments in the SSH
+    communicator. [GH-11909](https://github.com/hashicorp/packer/pull/11909)
+* core: During long running builds the HCP Packer registry will mark a build as
+    timed out if it has not posted an update in the last 10 minutes. For HCP
+    Packer enabled builds a status update will now be sent every 6 minutes to
+    the registry to prevent long builds from being marked as timed
+    out.[GH-11846](https://github.com/hashicorp/packer/pull/11846)
+* datasource/hcp_packer_image: Add `component_type` configuration argument to
```

Comment:
I think there's also the change that allows a channel to be configured directly on `hcp_packer_image`. Or was it released in the last version? 🤔

Reply:
Oh good questions. That might've got rolled into some other change and not called out. I will find it and add it if missing.

Reply:
Added.